### PR TITLE
Leave another comment in the documentation of SUNDIALS::IDA.

### PR DESCRIPTION
**Pull request by bangerth**

### CODE REVIEW

File: include/deal.II/sundials/ida.h [hunk 1]
```diff
@@ -274,6 +274,31 @@ namespace SUNDIALS
...
    * variables that evolves on a slower time scale.
    *
+   * Another case where we *could* eliminate a variable but do not want to
+   * is where that additional variable is introduced in the first place to work
+   * around some other problem. As an example, consider the time dependent
+   * version of the biharmonic problem we consider in step-47 (as well as some
+   * later ones). The equations we would then be interested in would read
+   * @f{align*}{
+   *   \frac{\partial u(\mathbf x,t)}{\partial t} + \Delta^2 u(\mathbf x,t) &=
+   *f(\mathbf x,t).
```
Comment (drwells) [inline include/deal.II/sundials/ida.h:284]:
```suggestion
   * f(\mathbf x,t).
```
This is rendered as `*f` in the generated doxygen - I think adding a space will fix it.

File: include/deal.II/sundials/ida.h [hunk 2]
```diff
@@ -274,6 +274,31 @@ namespace SUNDIALS
...
+   * @f}
+   * As discussed in step-47, the difficulty is the presence of the fourth
+   * derivatives. One way in which one can address this is by introducing
+   * an auxiliar variable $v=\Delta u$ which would render the problem into
```
Comment (drwells) [inline include/deal.II/sundials/ida.h:288]:
```suggestion
   * an auxiliary variable $v=\Delta u$ which would render the problem into
```

File: include/deal.II/sundials/ida.h [hunk 3]
```diff
@@ -274,6 +274,31 @@ namespace SUNDIALS
...
+   * the following one that only ever has second derivatives which we know
+   * how to deal with:
+   * @f{align*}{
+   *   \frac{\partial u(\mathbf x,t)}{\partial t} + \Delta v(\mathbf x,t) &=
+   *f(\mathbf x,t),
```
Comment (drwells) [inline include/deal.II/sundials/ida.h:293]:
```suggestion
   * f(\mathbf x,t),
```

**Review by drwells — APPROVED** [review]
Some minor formatting comments - can you fix them and merge?

**Review by marcfehling — APPROVED** [review]

**Pull request merged**